index.html

```diff
@@ -0,0 +1,22 @@
+<html>
+  <head>
+    <title>
+      Pats Salmon Goods
+    </title>
+    <link href='style.css' rel='stylesheet' type='text/css'>
+  </head>
+  <body>
+    <h1> Pats Salmon Cookies </h1>
+    <img id='salmon'src='https://raw.githubusercontent.com/codefellows/seattle-201d21/master/class-06-dom-objects-domain_modeling/lab/assets/salmon.png' alt='tasty salmon'>
+    <div>
+      customer reviews: 'The food is great!!!'
+      'Salmon cookies? Who thought they could be so good!'
+      '10 out of 10 so gooood'
+    <br />
+      business locations: 1st and Pike, SeaTac, Capitol Hill, Seattle Center, and Alki. We are open from 6am to 8pm daily.
+    <br />
+    contact info: 203-777-7777 or at patssalmoncookies@gmail.com
+  </div>
+  <script src='app.js'></script>
```

Comment:
none of the logic you're doing on app.js needs to be in index.html. You don't need this set of script tags.